projects/shr-mediator/src/common/services/fhir.service.ts

```diff
@@ -101,17 +118,29 @@ export class FhirService {
     options?: AxiosRequestConfig<any>,
     retryLimit = 2,
     timeout = 30000,
+    path?: string
   ) {
     for (let attempt = 1; attempt <= retryLimit; attempt++) {
       try {
-        const { data } = await this.httpService.axiosRef.post<R4.IBundle>(
-          config.get('fhirServer:baseURL'),
+        if(!path){
+          path=''
+        }
```

Comment:
Not sure this is correct

Reply:
To avoid 'undefined' exceptions

Reply:
Let's rather do this. 
`path?: string = ''`